.github/workflows/ci.yml

```diff
@@ -0,0 +1,88 @@
+name: CI
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    name: "Build"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        include:
+          - os: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build sourcemod plugin
+        uses: maxime1907/action-sourceknight@v1
+        with:
+          cmd: build
+
+      - name: Create package
+        run: |
+          mkdir -p /tmp/package
+          cp -R .sourceknight/package/* /tmp/package
```

Comment:
This line only adds the .smx to the final .tar.gz

There is also a translations folder to add so you can add these lines:
```
cp -R addons/sourcemod/translations /tmp/package/common/addons/sourcemod/
```